less/styles.less

```diff
@@ -41,24 +41,34 @@
     }
 
     h1 {
-      margin-bottom:10px;
-      margin-top:50px;
+      margin-bottom: 10px;
+      margin-top: 50px;
       &:after {
-        display:none;
+        display: none;
       }
     }
   }
 }
 
-.featured-side a{
-  float:none;
+.featured-side a {
+  float: none;
+}
+
+.featured-side .featured-side-content {
+  color: white;
+}
+
+.featured-story {
```

Comment:
If we ever introduce featured stories, this could break some of the functionality. I would advise just wrapping the featured content like such instead (which is the advised way of handling it via bootstrap):

```
<div class="row">
  <div class="featured-story ...">
    ... 
  </div>
</div>
```

Reply:
That's a good point, wrapping the content in a `<div class="row">` works like a charm!

By the way, is there a way to rebuild the stylesheet (I'm using `npm run production`) without restarting Hugo process?

I get:

```
lstat /path/to/asciidoc-wg.eclipse.org/node_modules/eclipsefdn-hugo-solstice-theme/static/mix.js: no such file or directory
lstat /path/to/asciidoc-wg.eclipse.org/node_modules/eclipsefdn-hugo-solstice-theme/static/mix.js: no such file or directory
hugo v0.82.0-9D960784 linux/amd64 BuildDate=2021-03-21T17:28:04Z VendorInfo=gohugoio

Reload Page
```

"Reload Page" does nothing... the only way is to restart `hugo server --destination=public` which is a bit annoying. Apart from that the development workflow is 👌🏻

Reply:
We found a way in the past, but I actually forget because I typically just rebuild. We can make a point in the theme to figure this out and document it since its a common issue though!

Reply:
Sounds good!
@autumnfound should I open an issue at https://github.com/EclipseFdn/hugo-solstice-theme?

Reply:
I've created one! https://github.com/EclipseFdn/hugo-solstice-theme/issues/215

Reply:
Perfect, thanks Martin!